### PR TITLE
Added login and registration components, styling

### PR DESCRIPTION
**Pull request by bethiamchan**
This branch has added components for login-view and registration-view. It has also added Bootsrap and SCSS styles.
image
image

### CODE REVIEW

**Comment by bethiamchan** [comment]
Additional functionality and styles have been added. Closing this pr to open a new one with updated code.

**Comment by bethiamchan** [comment]
The components now have improved functionality and styles are further improved.

### Login
image

### Registration
image

### Movie Cards
image

### Movie View
image

File: src/components/movie-card/movie-card.jsx [hunk 1]
```diff
@@ -1,13 +1,50 @@
...
 	render() {
 		const { movie, onClick } = this.props;
 
 		return (
-			<div onClick={() => onClick(movie)} className="movie-card">
-				{movie.Title}
-			</div>
+			<CardDeck className="movie-card-deck">
+				<Container>
+					<Card className="movie-card" style={{ width: '16rem', flex: 1 }} className="movie-card mb-3">
```
Comment (eadsimone) [inline src/components/movie-card/movie-card.jsx:17]: Please remove the duplicate attribute className

File: src/components/movie-view/movie-view.jsx [hunk 2]
```diff
@@ -8,35 +15,61 @@ export class MovieView extends React.Component {
...
-				</div>
-				<div className="back-to-main-view">
-					<button onClick={() => window.open('mainView', '_self')} className="back-button">
-						Back
-					</button>
-				</div>
+				<Container>
+					<Row>
+						<Col xs={3}>
+							<img className="movie-poster" src={movie.ImagePath} />
```
Comment (eadsimone) [inline src/components/movie-view/movie-view.jsx:27]: missing required "alt" attribute

**Review by eadsimone — APPROVED** [review]

**Pull request merged**